deployments/apim-developer.yml

```diff
@@ -0,0 +1,235 @@
+      AppServiceName: $(ApimDeveloperOuterApiAppServiceName)
+      DeploymentPackagePath: $(Pipeline.Workspace)/das-apim-endpoints-build/ApimEndpointsArtifacts/SFA.DAS.ApimDeveloper.Api.zip
+      ApiVersionSetName: $(ApimDeveloperOuterApiVersionSetName)
+      ApiPath: $(ApimDeveloperOuterApiPath)
+      ApiBaseUrl: $(ApimDeveloperOuterApiBaseUrl)
+      ProductId: $(ApimDeveloperProductId)
+      ApplicationIdentifierUri: $(ApimDeveloperOuterApiIdentifierUri)
+      AppRoleAssignmentsServiceConnection: das-app-role-assignments-CDS
+
+- stage: Deploy_TEST
+  dependsOn: []
+  displayName: Deploy to TEST
+  variables:
+  - group: DevTest Management Resources
+  - group: TEST DevTest Shared Resources
+  - group: TEST das-apim-endpoints
+  jobs:
+  - template: ../pipeline-templates/job/arm-deploy.yml
+    parameters:
+      ServiceConnection: SFA-DAS-DevTest-ARM
+      AppRoleAssignmentsServiceConnection: das-app-role-assignments-CDS
+      Environment: TEST
+      SchemaFilePath: $(ApimDeveloperOuterApiSchemaFilePath)
+
+  - template: ../pipeline-templates/job/deploy-api.yml
+    parameters:
+      DeploymentName: Deploy_ApimDeveloper
+      Environment: TEST
+      ServiceConnection: SFA-DAS-DevTest-ARM
+      AppServiceName: $(ApimDeveloperOuterApiAppServiceName)
+      DeploymentPackagePath: $(Pipeline.Workspace)/das-apim-endpoints-build/ApimEndpointsArtifacts/SFA.DAS.ApimDeveloper.Api.zip
+      ApiVersionSetName: $(ApimDeveloperOuterApiVersionSetName)
+      ApiPath: $(ApimDeveloperOuterApiPath)
+      ApiBaseUrl: $(ApimDeveloperOuterApiBaseUrl)
+      ProductId: $(ApimDeveloperProductId)
+      ApplicationIdentifierUri: $(ApimDeveloperOuterApiIdentifierUri)
+      AppRoleAssignmentsServiceConnection: das-app-role-assignments-CDS
+
+- stage: Deploy_TEST2
+  dependsOn: []
+  displayName: Deploy to TEST2
+  variables:
+  - group: DevTest Management Resources
+  - group: TEST2 DevTest Shared Resources
+  - group: TEST2 das-apim-endpoints
+  jobs:
+  - template: ../pipeline-templates/job/arm-deploy.yml
+    parameters:
+      ServiceConnection: SFA-DAS-DevTest-ARM
+      AppRoleAssignmentsServiceConnection: das-app-role-assignments-CDS
+      Environment: TEST2
+      SchemaFilePath: $(ApimDeveloperOuterApiSchemaFilePath)
+
+  - template: ../pipeline-templates/job/deploy-api.yml
+    parameters:
+      DeploymentName: Deploy_ApimDeveloper
+      Environment: TEST2
+      ServiceConnection: SFA-DAS-DevTest-ARM
+      AppServiceName: $(ApimDeveloperOuterApiAppServiceName)
+      DeploymentPackagePath: $(Pipeline.Workspace)/das-apim-endpoints-build/ApimEndpointsArtifacts/SFA.DAS.ApimDeveloper.Api.zip
+      ApiVersionSetName: $(ApimDeveloperOuterApiVersionSetName)
+      ApiPath: $(ApimDeveloperOuterApiPath)
+      ApiBaseUrl: $(ApimDeveloperOuterApiBaseUrl)
+      ProductId: $(ApimDeveloperProductId)
+      ApplicationIdentifierUri: $(ApimDeveloperOuterApiIdentifierUri)
+      AppRoleAssignmentsServiceConnection: das-app-role-assignments-CDS
+#
```

Comment:
I'd be tempted just to uncomment everything as it will all need approval before deploying anyways